rdbms/models/education.py

```diff
@@ -0,0 +1,48 @@
+import random
+
+from sqlalchemy import Column, Integer, Text, ForeignKey
+from sqlalchemy.exc import IntegrityError
+
+from .database import Base
+from .database import session
+from .users import Users
+
+class Education(Base):
+
+    __tablename__='education'
+
+    id = Column(Integer, nullable=False, primary_key=True)
+    user_id = Column(Integer, ForeignKey('users.id'))
+    school = Column(Text)
```

Comment:
Use String for short sequences of variable characters. Text is intended for long-form content such as articles.